app/src/features/apiClient/screens/PostmanImporterView/components/PostmanImporter/utils.ts

```diff
@@ -82,21 +82,21 @@ const processScripts = (item: any) => {
   return scripts;
 };
 
-const processAuthorizationOptions = (item = {}) => {
+const processAuthorizationOptions = (item: Record<string, any> = {}, parentCollectionId: string): RQAPI.AuthOptions => {
   try {
-    const auth = {};
-    const authType = POSTMAN_AUTH_TYPES_MAPPING[item.type] ?? AUTHORIZATION_TYPES.NO_AUTH;
-    auth.currentAuthType = authType;
-    auth[authType] = {};
-
-    const authOptionsArray = item[item?.type] || [];
-    authOptionsArray.forEach((option) => {
-      auth[authType][option.key] = option.value;
+    const currentAuthType =
+      POSTMAN_AUTH_TYPES_MAPPING[item.type] ??
+      (parentCollectionId ? AUTHORIZATION_TYPES.INHERIT : AUTHORIZATION_TYPES.NO_AUTH);
```

Comment:
this has edge cases for when imported auth type is `Inherit` and `no_auth` since they are not part of `POSTMAN_AUTH_TYPES_MAPPING`, please update that.

the two edge cases: 
1. if a request inside a collection (_has a parent Id_) and the request's auth inside postman is `no_auth`, then it will be set to `inherit` when importing here
2. if a top level collection has auth as `inherit` in postman it will get converted to `no_auth`  when imported to requestly